spec/top_secret/constants_spec.rb

```diff
@@ -14,4 +14,27 @@
       expect(phone_number).to match(TopSecret::PHONE_REGEX)
     end
   end
+
+  bad_phone_numbers = [
+    "555) 555-5555",
+    "(555 555-5555",
+    "415+555+1234",
+    "(555 ) 555-5555",
+    ")555( 555-5555"
+  ]
```

Comment:
Note: the previous `PHONE_REGEX` matched the first 3 `bad_phone_numbers`.  The last 2 were added as part of working through an updated regex.